js/tools/path.js

```diff
@@ -120,8 +123,18 @@ class Path extends Shape {
         canvas.canvas.removeEventListener("mousemove", Path.onMouseMoveNode);
     }
 
+    static onMouseEnterNode(event) {
+        event.target.setAttributeNS(null, "r", 6);
```

Comment:
what's `r`

Reply:
`r` is the radius of the node (which is a circle internally)